dictionary/MessageFormatDictionary.js

```diff
@@ -80,22 +80,66 @@ export default class MessageFormatDictionary extends Dictionary {
 	 *        Defaults to an empty plain object.
 	 */
 	get(key, parameters = {}) {
+		const scope = this._getScope(key);
+
+		if (scope === null) {
+			throw new GenericError(
+				`ima.dictionary.MessageFormatDictionary.get: The ` +
+				`localization phrase '${key}' does not exists`,
+				{ key, parameters }
+			);
+		}
+
+		return scope(parameters);
+	}
+
+	/**
+	 * @inheritdoc
+	 * @param {string} key The key identifying the localization phrase. The key
+	 *        consists of at least two parts separated by dots. The first part
+	 *        denotes the name of the source JSON localization file, while the
+	 *        rest denote a field path within the localization object within
+	 *        the given localization file.
+	 */
+	has(key) {
+		if (!/^[^.]+\.[^.]+$/.test(key)) {
+			throw new Error(
+				`The provided key (${key}) is not a valid localization phrase key, ` +
```

Comment:
This line is 86 characters long (tabs are considered to be 4 characters - spaces - wide), please reformat this.